### ISSUE TITLE
P0: Fix accessibility in `Collapse` component

### ISSUE THREAD

**Issue by yaelleC** [grafana/grafana#56558]
We noticed two keyboard a11y issues with the resource picker:

- [ ] Advanced view in resource picker is not keyboard accessible (you cannot tab into it)
The issue here lies with the `Collapse` component and the best way forward would be to fix it for all the places it's used
- [x] ~~The resource picker modal doesn't trap the focus state correctly, e.g. you can get into a state where you're tabbing behind the modal~~ Removing since it's a known limitation (#43974)

**Comment by andresmgot** [comment]
> * The resource picker modal doesn't trap the focus state correctly, e.g. you can get into a state where you're tabbing behind the modal

This is intentional, see https://github.com/grafana/grafana/pull/43974 :disappointed_relieved:

**Comment by yaelleC** [comment]
Closing as this was solved in 9.3 😄
